src/components/textbox/textbox.spec.js

```diff
@@ -321,4 +312,72 @@ describe("Textbox", () => {
       });
     });
   });
+
+  describe("new validations", () => {
+    const renderWithNewValidations = ({ error, warning }) =>
+      mount(
+        <CarbonProvider validationRedesignOptIn>
+          <Textbox
+            labelHelp="Example hint text"
+            error={error}
+            warning={warning}
+            labelAlign="left"
+            labelInline
+            labelWidth={100}
+            reverse
+          />
+        </CarbonProvider>
+      );
+
+    describe("label width and align props", () => {
+      it("default to undefined", () => {
+        const wrapper = renderWithNewValidations({});
+        const { labelAlign, labelInline, reverse, labelWidth } = wrapper
+          .find(FormField)
+          .props();
+
+        expect(labelAlign).toEqual(undefined);
+        expect(labelInline).toEqual(undefined);
+        expect(reverse).toEqual(undefined);
+        expect(labelWidth).toEqual(undefined);
+      });
+    });
+
+    describe("hint/ labelHelp", () => {
```

Comment:
Is this describe description a mistake or should the forward slash be there?

Reply:
They are the same thing, the hint uses the `labelHelp` as its text, I can update it if you want I think I just left it the way Rafal had it tbh